main.go

```diff
@@ -77,9 +77,7 @@ func main() {
 
 	if os.Getenv("GITHUB_WORKSPACE") != "" {
 		// inside a Github action, export vulns
-		if output, err := security.Format(vulns, "raw_json"); err != nil {
```

Comment:
I would replace `raw_json` by `json` instead

Reply:
If we do that, it will not work because json formatter will output a json on more than one line (like my solution), and after some search we need to replace `\n` `\r` and `%` chars to have a multiline output on github action.

With these informations, I think the best choice was to implement a raw_json formatter that will output a one line json.

Reply:
I have added a pr to add a raw json formatter. I am a little unsure if this check would also need to be changed to an == instead of a !=
"err != nil {"

Reply:
If you're adding the raw_json formatter this line should stay as raw_json